src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+@import "../utils/variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position:fixed;
```

Comment:
it should be neither fixed nor absolute

just use margins to place it correctly